sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.UnitTest/HttpParsers/AzureBlobHttpParserTests.cs

```diff
@@ -0,0 +1,175 @@
+                Tuple.Create("Lease Blob",                       "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=lease",                                        "myaccount", blobProperties),
+                Tuple.Create("Snapshot Blob",                    "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=snapshot",                                     "myaccount", blobProperties),
+                Tuple.Create("Copy Blob",                        "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob",                                                   "myaccount", blobProperties),
+                Tuple.Create("Abort Copy Blob",                  "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=copy&copyid=id",                               "myaccount", blobProperties),
+                Tuple.Create("Put Block",                        "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=block&blockid=id",                             "myaccount", blobProperties),
+                Tuple.Create("Put Block List",                   "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=blocklist",                                    "myaccount", blobProperties),
+                Tuple.Create("Get Block List",                   "GET",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=blocklist",                                    "myaccount", blobProperties),
+                Tuple.Create("Get Block List",                   "GET",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=blocklist&snapshot=DateTime",                  "myaccount", blobProperties),
+                Tuple.Create("Put Page",                         "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=page",                                         "myaccount", blobProperties),
+                Tuple.Create("Get Page Ranges",                  "GET",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=pagelist",                                     "myaccount", blobProperties),
+                Tuple.Create("Get Page Ranges",                  "GET",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=pagelist&snapshot=DateTime",                   "myaccount", blobProperties),
+                Tuple.Create("Get Page Ranges",                  "GET",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=pagelist&snapshot=DateTime&prevsnapshot=Date", "myaccount", blobProperties),
+                Tuple.Create("Append Block",                     "PUT",      "https://myaccount.blob.core.windows.net/my/container/myblob?comp=appendblock",                                  "myaccount", blobProperties)
```

Comment:
this is fine, but consider using XUnit.Theory. 
The main benefit is it will tell you specifically which scenario passes/fails.